cpp/src/Ice/ConnectionI.cpp

```diff
@@ -1083,7 +1083,7 @@ Ice::ConnectionI::setCloseCallback(ICE_IN(ICE_DELEGATE(CloseCallback)) callback)
                 CallbackWorkItem(const ConnectionIPtr& connection, ICE_IN(ICE_DELEGATE(CloseCallback)) callback) :
                     _connection(connection),
 #ifdef ICE_CPP11_MAPPING
-                    _callback(move(callback))
+                    _callback(std::move(callback))
```

Comment:
That's fine but why do we need this `std::`? There is ambiguity with another move function?

Reply:
I would keep it, I find the `std::move` more readable.